data/cpu_bugs/vm_install_script/sle-15/create_vm_url.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [ $# -lt 4 ]; then
+    echo "$0 <name> <install_url> <autoyast_url> <logfile_path> <vm_pool> [cpu]"
+    exit 1
+fi
+
+NAME=$1
+QCOW2POOL=$5
+LOGFILE=$4
+INSTALL_URL=$2
+AUTOYAST_URL=$3
+CPU=$6
+if [ -z $CPU ]; then
+    CPU=host-model-only
+fi
+
+mkdir -pv $QCOW2POOL
+
+virt-install --name ${NAME} \
+    --disk path=$QCOW2POOL/"${NAME}.qcow2",size=20,format=qcow2,bus=virtio,cache=none \
+    --os-variant sle15 \
+    --noautoconsole \
+    --wait=-1 \
+    --vnc \
+    --vcpus=4 \
+    --cpu ${CPU}\
+    --ram=1024 \
+    --console=log.file=${LOGFILE}\
+    --network bridge=br0,model=virtio \
+    --location=${INSTALL_URL} \
+    -x "console=ttyS0,115200n8
+        install=${INSTALL_URL}
+	autoyast=${AUTOYAST_URL}"
+
+
```

Comment:
In general for shellscripts I recommend to use a linting tool, e.g. try "shellcheck" from the package "ShellCheck". What I can already see as a generic suggestion for this file and others then as well:
Use `!#/bin/sh -e` unless you actually need bash features, then use `#!/bin/bash -e`

Reply:
OK, I will check it.